next.config.js

```diff
@@ -0,0 +1,44 @@
+let config = {
+  typescript: {
+    // !! WARN !!
+    // Dangerously allow production builds to successfully complete even if
+    // your project has type errors.
+    // !! WARN !!
+    ignoreBuildErrors: true
+  },
+  async rewrites() {
+    return [
+      {
+        source: "/api/configs_db/:path*",
+        destination:
+          "https://incident-commander.canary.lab.flanksource.com/config/db/:path*"
+      },
+      {
+        source: "/api/canary/:path*",
+        destination:
+          "https://incident-commander.canary.lab.flanksource.com/canary/:path*"
+      },
+      {
+        source: "/api/incidents_db/:path*",
```

Comment:
Maybe /api/db/ ?

Reply:
There is also config_db, and canary etc. Can rename if it still makes sense. Thought it was better to be explicit about which db it is.